app/src/main/java/org/fossasia/openevent/app/contract/model/UtilModel.java

```diff
@@ -2,10 +2,20 @@
 
 public interface UtilModel {
```

Comment:
Secondly, UtilModel classes depend on context and thus must be abstracted. Holding context in a static variable is a serious memory leak